addons/im_livechat/models/mail_channel.py

```diff
@@ -76,3 +76,11 @@ def cron_remove_empty_session(self):
                 < ((now() at time zone 'UTC') - interval %s)""", ("%s hours" % hours,))
         empty_channel_ids = [item['id'] for item in self.env.cr.dictfetchall()]
         self.browse(empty_channel_ids).unlink()
+
+    @api.model
+    def get_empty_list_help(self, help):
+        if help:
+            action_channel = self.env.ref('im_livechat.mail_channel_action', False)
```

Comment:
If we remove this then for all mail.channel action inside im_chatter , we just override help message. is it ok ?